rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1196,3 +1196,40 @@ def test_events_filter_params(rotkehlchen_api_server, ethereum_accounts):
     result = assert_proper_response_with_result(response)
     expected = generate_tx_entries_response([(tx1, [event3])])
     assert result['entries'] == expected
+
+
+def test_ignored_assets(rotkehlchen_api_server, ethereum_accounts):
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    db = rotki.data.db
+    db.add_to_ignored_assets(A_BTC)
+    dbethtx = DBEthTx(db)
+    tx1 = make_ethereum_transaction(tx_hash=b'1')
+    tx2 = make_ethereum_transaction(tx_hash=b'2')
+    event1 = make_ethereum_event(tx_hash=b'1', index=1, asset=A_ETH)
```

Comment:
Add 1 more transaction with a different asset and then add a second ignored asset in the test to make sure this works for multiple ignored assets.

Reply:
Alright, adjusted tests to test multiple ignored assets